dts/bindings/test/vnd,memory-attr.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 Linaro Ltd.
```

Comment:
Copyright

Reply:
aaaargh, not going to push again and lose all the ACKs for that :(

Reply:
> aaaargh, not going to push again and lose all the ACKs for that :(

Me and @erwango can do another +1 quickly if you want to push and get the credit, but it's up to you. 😉

Reply:
nah, let Linaro have the cake and eat it too :smile: 

I can eventually fix that later.